projects/js-packages/charts/src/components/legend/legend.module.scss

```diff
@@ -0,0 +1,36 @@
+.legend {
+	&--horizontal {
+		display: flex;
+		flex-direction: row;
+		flex-wrap: wrap;
+		gap: 16px;
+	}
+
+	&--vertical {
+		display: flex;
+		flex-direction: column;
+		gap: 8px;
+	}
+}
+
+.legend-item {
+	display: flex;
+	align-items: center;
+	gap: 8px;
+	font-size: 0.875rem;
+}
+
+.legend-item-swatch {
+	border-radius: 2px;
+}
+
+.legend-item-label {
+	color: var(--jp-gray-80);
```

Comment:
Let's add a fallback in case `--jp` is not available `var(--jp-gray-80, #ZZZZZZ)`